webpack.common.js

```diff
@@ -1,30 +1,45 @@
 const { ESBuildPlugin, ESBuildMinifyPlugin } = require('esbuild-loader');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
-module.exports = {
-  resolve: {
-    alias: {
-      'video.js': 'video.js/core',
+module.exports = (env) => {
```

Comment:
wrap this in a function that accepts the `env` object. Easiest to review this file by hiding whitespace.